### DISCUSSION TITLE
Pi sending Midi Clock to Teensy?

### DISCUSSION THREAD

**Discussion by steven-law** [FundamentalFrequency/.github#46] [Ideas]
Hey,
does the Pi sends Midi Clock to the teensy?
I could think of some fun applications like Arpeggiators, Sequencer and Stuff.
As we know the teensy is quite overpowered for what it does right now,
so i think, maybe we could cram some funny Apps into it.
I´m new but familiar with programming USBMidi stuff for the teensy.

**Comment by stonepreston** [comment]
the pi does not currently send clock to the teensy. It does seem this is possible with the tracktion engine as Dave [states in this forum post](https://forum.juce.com/t/sync-to-midi-clock/50772). So there is potential for that, if youd like to try.
